presto-main/src/main/java/com/facebook/presto/operator/PagesRTreeIndex.java

```diff
@@ -155,11 +112,11 @@ private static Envelope getEnvelope(OGCGeometry ogcGeometry)
 
         IntArrayList matchingPositions = new IntArrayList();
 
-        Envelope envelope = getEnvelope(probeGeometry);
-        rtree.query(envelope, item -> {
+        Rectangle queryRect = GeometryUtils.getExtent(probeGeometry);
```

Comment:
static import + inline: `rtree.findIntersections(getExtent(probeGeometry), item -> {`

Reply:
We can't inline, because it's used in the closure (for `testReferencePoint`).